osf/management/commands/sync_citation_styles.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand
+import re
+import os
+import io
+import zipfile
+import requests
+import logging
+from django.apps import apps
+from api.base import settings
+
+from django.db import transaction
+from lxml import etree
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+
+
+def sync_citation_styles(dry_run=False):
```

Comment:
* This is a command version of  `update_styles(state, schema):` in migration `0107_add_dependent_styles.py`. It would be great if you can mention the changes you applied in PR descriptions since GitHub is not smart when we move things around. Here is a few changes I discovered but I might have missed others.
  * Loading styles from a file vs from a online resource
  * Hard-coded the online resource
  * Some readability improvements for names such as `ns` -> `namespace`
  * Added dry-run option

* What happens to `parse_citation_styles(state, schema)` in migration `0074_parse_citation_styles.py`? Maybe another command? Or `parse_citation_styles(state, schema)` was deprecated and replaced by `update_styles(state, schema)`?